site/content/en/docs/manual/advanced/dataset_manifest.md

```diff
@@ -11,11 +11,62 @@ description: This section on [GitHub](https://github.com/cvat-ai/cvat/tree/devel
 
 <!--lint disable heading-style-->
 
-### Steps before use
+## Overview
 
-When used separately from Computer Vision Annotation Tool(CVAT), the required dependencies must be installed
+When a new task is created in CVAT, we need to tell where to take the dataset sources.
+CVAT allows to use different data sources, including file uploads, a mounted server
+file share, cloud storage files and others.
 
-#### Ubuntu:20.04
+Dataset manifest files in CVAT allow to provide extra information about the source data.
+They are mainly used when working with cloud storages to reduce the amount
+of network traffic used and speed up the task creation process. However, they can also
+be used in other cases.
+
+A manifest file is a text file in the JSONL format.
+
+## How and when to use manifest files
+
+Manifest files can be used in the following cases:
+- A video file or a set of images is used as the data source and
+  the caching mode is enabled (`use_cache`). [Read mode](/docs/manual/advanced/data_on_fly/)
```

Comment:
What is `use_cache`? It's not clear from this text. The linked page doesn't explain it either.

Reply:
Updated, I tried to make it a bit clearer.